ecf/defs/evs-nco.def

```diff
@@ -875,6 +927,60 @@ suite evs_nco
             task jevs_analyses_rtma_ru_grid2obs_stats_vhr_11
               time 11:00
               edit VHR '11'
+            task jevs_analyses_rtma_precip_stats_vhr_06
+              time 06:00
+              edit VHR '06'
+            task jevs_analyses_rtma_precip_stats_vhr_07
+              time 07:00
+              edit VHR '07'
+            task jevs_analyses_rtma_precip_stats_vhr_08
+              time 08:00
+              edit VHR '08'
+            task jevs_analyses_rtma_precip_stats_vhr_09
+              time 09:00
+              edit VHR '09'
+            task jevs_analyses_rtma_precip_stats_vhr_10
+              time 10:00
+              edit VHR '10'
+            task jevs_analyses_rtma_precip_stats_vhr_11
+              time 11:00
+              edit VHR '11'
+            task jevs_analyses_urma_precip_stats_vhr_06
+              time 06:00
+              edit VHR '06'
+            task jevs_analyses_urma_precip_stats_vhr_07
+              time 07:00
+              edit VHR '07'
+            task jevs_analyses_urma_precip_stats_vhr_08
+              time 08:00
+              edit VHR '08'
+            task jevs_analyses_urma_precip_stats_vhr_09
+              time 09:00
+              edit VHR '09'
+            task jevs_analyses_urma_precip_stats_vhr_10
+              time 10:00
+              edit VHR '10'
+            task jevs_analyses_urma_precip_stats_vhr_05
```

Comment:
Please check line 963. I think it needs to end in `_vhr_11`

Reply:
Fixed.